main.go

```diff
@@ -88,9 +88,24 @@ func main() {
 		exit(1)
 	}
 
-	adds, deletes := localRecords.Diff(records)
+	// Cast the records from Cloudflare to a recordCollection.
```

Comment:
This comment just explains what the next line does. I'd rather have one explaining why, or how it's done. Actually, in this case, I'd rather there was no comment.